### PR TITLE
Windexer

### PR DESCRIPTION
**Pull request by c0d3rman**
Added the windexer and made some flywheel changes.

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/commands/FlywheelSpinup.java [hunk 1]
```diff
@@ -3,10 +3,16 @@
 
 import org.usfirst.frc4904.robot.RobotMap;
 import org.usfirst.frc4904.robot.subsystems.Flywheel;
-import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import org.usfirst.frc4904.standard.commands.motor.MotorSet;
 
-public class FlywheelSpinup extends MotorConstant {
```
Comment (carturn) [inline src/org/usfirst/frc4904/robot/commands/FlywheelSpinup.java:8]: This is a motor set being hacked to act like a motor constant.

File: src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java [hunk 2]
```diff
@@ -31,11 +35,16 @@ public DefaultOperator(String name) {
...
-		RobotMap.Component.operatorStick.button5.whenReleased(new RampSet(GearIO.RampState.EXTENDED));
+		RobotMap.Component.operatorStick.button3.onlyWhileHeld(new FlywheelSpinup());
+		RobotMap.Component.operatorStick.button4.whenPressed(new RampSet(GearIO.RampState.RETRACTED));
+		RobotMap.Component.operatorStick.button4.whenReleased(new RampSet(GearIO.RampState.EXTENDED));
+		RobotMap.Component.operatorStick.button5.onlyWhileHeld(new IndexerLoad());
 		RobotMap.Component.operatorStick.button7.onlyWhileHeld(new GearClear());
 		RobotMap.Component.operatorStick.button9.whenPressed(new RampSet(GearIO.RampState.RETRACTED));
 		RobotMap.Component.operatorStick.button10.whenPressed(new RampSet(GearIO.RampState.EXTENDED));
+		RobotMap.Component.operatorStick.button12
+			.whenPressed(
```
Comment (carturn) [inline src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java:46]: Can we just add an offset to a motor set?

**Review by carturn — CHANGES_REQUESTED** [review]
I would replace the motor constant with changing constant thing with a motor set extended to have an offset.

File: src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java (hunk 2 above)
Comment (c0d3rman) [reply src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java:46]: How would that work with a MotorConstant?

**Review by carturn — APPROVED** [review]

**Comment by carturn** [comment]
Merging for now because competition.

**Comment by carturn** [comment]
Leave the branch for now.

File: src/org/usfirst/frc4904/robot/commands/FlywheelSpinup.java (hunk 1 above)
Comment (ajnadel) [reply src/org/usfirst/frc4904/robot/commands/FlywheelSpinup.java:8]: ...by means of a mutable "constant" (`Flywheel.SHOOTING_SPEED`).

**Pull request merged**